demos/using-dev-build/webapp/index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <style>
+      :root {
+      }
+
+      body {
+        margin: 0;
+        font-family: Helvetica, Arial, Sans-Serif;
+      }
+
+      img {
+      }
+
+      main {
+        max-width: 800px;
+        margin: 0 auto;
+        text-align: center;
+      }
```

Comment:
```
input {
  width: 24rem;
}
```

<img width="538" alt="Screenshot 2022-04-07 at 14 31 56" src="https://user-images.githubusercontent.com/98735375/162203608-33cef5ae-73f3-4d48-9afb-485990c23669.png">

Reply:
Good call, set `size` to `50` on the `label`.